### ISSUE TITLE
tf.nn.dropout()

### ISSUE THREAD

**Issue by shibin2018** [Google-Health/records-research#4]
In the train.py,the 202th line tf.nn.dropout(rate=...),but the tf.nn.dropout() has no 'rate' parameter

**Comment by jonasbkemp** [comment]
@mp2893 

Looks like the 'rate' parameter was added to tf.nn.dropout in TF 1.13. Try a more recent version of TF1. We'll update the docs accordingly, thanks for the report.

**Comment by shibin2018** [comment]
thank you very much. I want to ask that how to train this model with the synthetic_samples.I find that you only give the graph-convolutional-transformer code with the eICU dataset. Because I can't download the eICU dataset,I want to use the synthetic dataset.
----- 原始邮件 -----
发件人：Jonas Kemp <notifications@github.com>
收件人：Google-Health/records-research <records-research@noreply.github.com>
抄送人：shibin2018 <shibin2014@sina.com>,  Author <author@noreply.github.com>
主题：Re: [Google-Health/records-research] tf.nn.dropout() (#4)
日期：2020年04月22日 02点44分


@mp2893
Looks like the 'rate' parameter was added to tf.nn.dropout in TF 1.13. Try a more recent version of TF1. We'll update the docs accordingly, thanks for the report.

**Comment by jonasbkemp** [comment]
Updated documented TF version in commit c89c27d311f4e61e5bc1edc33ab7eeefe5eac8a7.
